ui/src/VoxettaUserForm.js

```diff
@@ -0,0 +1,255 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+ 
+import {LitElement, html, css} from 'lit-element';
+
+import {CookieService} from './CookieService';
+import {TextField} from '@material/mwc-textfield';
+import {Select} from '@material/mwc-select';
+import {ListItem} from '@material/mwc-list/mwc-list-item';
+import {Button} from '@material/mwc-button';
+import {Icon} from '@material/mwc-icon'
+ 
+/**
+ * Component responsible for providing users a means to provide
+ * their personal information. 
+ */
+export class VoxettaUserForm extends LitElement {
+
+    static get styles() {
+        return css`
+            .container {
+                align-items: center; 
+                color: #3c4043;
+                display: flex; 
+                flex-direction: column; 
+                flex-wrap: wrap;
+                justify-content: center; 
+                text-align: center;  
+                width: 400px; 
+            }
+            .header-container {
+                align-items: center;
+                display: flex; 
+                flex-direction: row; 
+                flex-wrap: wrap;
+                justify-content: center; 
+                width: 270x; 
+            }
+            .title-container {
+                width: 250px; 
+            }
+            .icon-container {
+                width: 20px; 
+            }
+            h1, h2 { 
+                font-family: 'Roboto';
+                font-weight: normal; 
+                margin-top: 5px;
+                margin-left: 0px;
+                margin-right: 0px; 
+                width: 270px; 
+            }
+            h1.title {
+                font-size: 30px;
+                margin-bottom: 5px; 
+                padding-left: 5px; 
+            }
+            h2.description {
+                font-size: 18px;
+                margin-bottom: 10px;
+            }
+            mwc-icon {
+                --mdc-icon-size: 40px; 
+                float: left;
+                position: relative;
+            }
+            mwc-textfield {
+                --mdc-theme-primary: #1a73e8;
+                padding: 10px; 
+                width: 300px;
+            }
+            mwc-select {
+                --mdc-theme-primary: #1a73e8;
+                padding: 10px; 
+                width: 300px;
+            }
+            mwc-button {
+                padding: 10px;
+                width: 250px; 
+            }
+            mwc-button.save {
+                --mdc-theme-primary: #1a73e8;
+                --mdc-theme-on-primary: white;
+            }
+            mwc-button.cancel {
+                --mdc-theme-primary: white;
+                --mdc-theme-on-primary: #1a73e8;
+            }
+        `;
+    }
+
+    constructor() {
+        super();
+        this.cookieService = new CookieService();
```

Comment:
Pass these information in from property bindings, not from the cookie. The `vox-app` knows about cookies, but `vox-user-form` doesn't necessarily need to, it can receive this information from the parent.

https://lit-element.polymer-project.org/guide/properties

And conversely, `vox-user-form` can communicate updates back up to parent by dispatching custom events.

https://lit-element.polymer-project.org/guide/events#fire-event

The goal is to minimize responsibilities of each component. Here, we want to isolate cookie dependency to only `vox-app` if possible, and make `vox-user-form` only care about form validation and input management.

Reply:
adjusted